functions/ts/src/sops.ts

```diff
@@ -0,0 +1,206 @@
+import {
```

Comment:
can you add some e2e tests for this to ensure it doesn't bit rot?

Reply:
sure - I can :) . I wasn't sure if you were interested in this function. Now when I see the interest, I will make e2e based on the already existing tests for other functions, along with resolving other comments.

Reply:
added tests/sops.sh